addons/pos_self_order/static/src/app/components/product_card/product_card.js

```diff
@@ -7,7 +7,7 @@ import { Line } from "@pos_self_order/app/models/line";
 
 export class ProductCard extends Component {
     static template = "pos_self_order.ProductCard";
-    static props = ["product", "currentProductCard?"];
+    static props = ["product", "currentProductCard?", "onProductInfoClick?"];
```

Comment:
Hello, you don't need to pass a props, put the function directly in product_card